wiki/MapData.md

```diff
@@ -174,7 +177,13 @@ Systems are the locations that ships are capable of being and can contain planet
 system <name>
 ```
 
-The name of a system must be unique.
+The true name of a system must be unique. Use this name to internally reference systems in missions, events, etc.
+
+```html
+"display name" <name>
+```
+
+Added in **v. 0.10.11**, this name is displayed to the player. If not defined, it defaults to the true name of the system. Multiple systems can share the same display name.
```

Comment:
```suggestion
Since **v. 0.10.11**: Specifies the name displayed to the player. If not defined, it defaults to the true name of the system. Multiple systems can share the same display name.
```